test/e2e/serial/tests/workload_placement.go

```diff
@@ -809,3 +813,30 @@ func matchLogLevelToKlog(cnt *corev1.Container, level operatorv1.LogLevel) (bool
 	val, found := rteFlags.GetFlag("--v")
 	return found, val.Data == kLvl.String()
 }
+
+func logSchedulerPluginLogs(fxt e2efixture.Fixture) {
+	nroSchedObj := objects.TestNROScheduler()
+	err := e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)
+	if err != nil {
+		klog.Warningf("error getting the scheduler plugin CR: %v", err)
+		return
+	}
+	schedDp, err := schedutils.GetDeploymentByOwnerReference(nroSchedObj.GetUID())
+	if err != nil {
+		klog.Warningf("error getting the scheduler deployment: %v", err)
+		return
+	}
+	schedPods, err := schedutils.ListPodsByDeployment(fxt.Client, *schedDp)
+	if err != nil {
+		klog.Warningf("error getting the scheduler pod: %v", err)
+		return
+	}
+	Expect(len(schedPods)).To(Equal(1))
```

Comment:
the code above (never returning error, using klog and no expectations) suggests we want to tolerate failures in this function and not abort the test should this function fail. If so, why adding this only `Expect`?

Reply:
right, same I thought about, but having more than one scheduler pod can indicate a serious problem here.

Reply:
Yet, it is not out goal in this function to check that so I'll make it and if with warning.